src/main/scala/is/hail/HailContext.scala

```diff
@@ -501,22 +501,28 @@ class HailContext private(val sc: SparkContext,
   }
 
   def importMatrix(file: String,
+    nAnnotations: Int = 1,
```

Comment:
Whoa, I didn't know you could do this:

```
scala> def f(i: Int = 5, j: Int): Int = i + j
f: (i: Int, j: Int)Int

scala> f(j=2)
res0: Int = 7
```

Anyway, seems weird but going to get deleted.